packages/terra-clinical-item-collection/src/_ItemCollectionUtils.jsx

```diff
@@ -102,8 +102,8 @@ function createOnSelectProps(onSelect, key) {
     onSelect(event, key);
   };
 
-  selectableProps.onKeyDown = (event) => {
-    if (event.nativeEvent.keyCode === KEYCODES.ENTER || event.nativeEvent.keyCode === KEYCODES.SPACE) {
+  selectableProps.onKeyDown = (event) => { 
+    if (event.nativeEvent.keyCode === KeyCode.KEY_ENTER || event.nativeEvent.keyCode === KeyCode.KEY_SPACE) {
```

Comment:
We should add an import at the top of this file to import the keycode's module.

```diff
+ import KeyCode from 'keycode-js';
```

Then, we should be able to remove the `KEYCODES` const [defined at the top of this file](https://github.com/cerner/terra-clinical/pull/479/files#diff-c12336a4490f4dce5ed72463a37231f3R1).

Reply:
This import is already there. https://github.com/cerner/terra-clinical/pull/479/files#diff-c12336a4490f4dce5ed72463a37231f3R1